src/fenics_constitutive/solver.py

```diff
@@ -289,14 +296,11 @@ def update(self) -> None:
         self.stress_0.x.scatter_forward()
 
         for k, (law, _) in enumerate(self.laws):
+            law.update()
```

Comment:
This is mainly a feature for the Abaqus UMAT-implementation in the paper. I would always prefer the constitutive model classes to not store any data, but this may not always be possible and therefore we have the `update` method that can be called to update the internally stored data. But we just forgot to use this in our code previously because none of our models uses this feature.

Reply:
Understood